lib/config.js

```diff
@@ -95,26 +95,36 @@ function loadConfig(filePath) {
         // a "parent". Load the referenced file and merge the configuration recursively.
         if (config.extends) {
 
-            var parentPath = config.extends;
-            if (isFilePath(parentPath)) {
-                // If the `extends` path is relative, use the directory of the current configuration
-                // file as the reference point. Otherwise, use as-is.
-                parentPath = (!isAbsolutePath(parentPath) ?
-                    path.join(path.dirname(filePath), parentPath) :
-                    parentPath
-                );
+            if (!Array.isArray(config.extends)) {
+                config.extends = [config.extends];
             }
 
-            // Merge the configuration, ensuring children get preference over the parent
-            try {
-                config = util.mergeConfigs(loadConfig(parentPath), config);
-            } catch (e) {
-                // If the file referenced by `extends` failed to load, add the path to the
-                // configuration file that referenced it to the error message so the user is
-                // able to see where it was referenced from, then re-throw
-                e.message += "\nReferenced from: " + filePath;
-                throw e;
-            }
+            // Make the last element in an array take the highest precedence
+            config.extends = config.extends.reverse();
```

Comment:
Look at the way the test fixtures are structured, and what is being expected with the overrides.